sdk/tcp_listener.go

```diff
@@ -14,10 +14,16 @@ const (
 )
 
 func newTCPListener(address string, pluginName string) (net.Listener, string, error) {
-	listener, err := sockets.NewTCPSocket(address, nil)
+	listener, err := setupSocketActivation()
```

Comment:
I mean the listen stream directive in the socket file can be 0.0.0.0:8080 for instance